clj/src/cljd/core.cljd

```diff
@@ -8760,3 +8760,48 @@ specified.
   ([aseq]
    (.from #/(List dynamic) aseq .growable false))
   ([type aseq] (into-array aseq)))
+
+(defmacro time
+  "Evaluates expr and prints the time it took.  Returns the value of expr."
+  {:added "1.0"}
+  [expr]
+  `(let [stopwatch# (dart:core/Stopwatch)
+         _ (.start stopwatch#)
```

Comment:
`_` is no magic in macros and should be an autogensym like other locals. So `_#`.
But this could be reworked to get rid of them altogether: the start call can be bundled with the constructor using doto and the second one may move to the let body.

Reply:
I think `_#` may be the only viable path. I tried added `doto`, however it cause a dynamic warning, even with type hints.

Reply:
Smells like a bug (doto -> dynamic warning) then.